GitOut/Features/Wpf/Converters/NullToVisibilityMultiConverter.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Windows;
+using System.Windows.Data;
+
+namespace GitOut.Features.Wpf.Converters
+{
+    public class NullToVisibilityMultiConverter : IMultiValueConverter
+    {
+        public object Convert(object[] values, Type targetType, object parameter, CultureInfo culture) => values.Any(v => v != null) ? Visibility.Visible : Visibility.Collapsed;
```

Comment:
`!(is null)` instead of !=
Or we could use the inverse `values.Length == 0 || values.All(v => v is null) ? Collapsed : Visible`

Reply:
note that `!= null` has been used at multiple locations.

Reply:
Yep, was more of a suggestion